process.sh

```diff
@@ -37,6 +37,7 @@ main() {
     else
         aws s3 cp --exclude "$OUTPUT/*.ome.tif*" --exclude "$OUTPUT/*.zarr/*" --recursive "$OUTPUT" s3://"$CLOUD_TARGET"
         gsutil -m cp -r "$OUTPUT/vanderbilt/vanderbilt.images" "gs://$CLOUD_TARGET/vanderbilt/vanderbilt.images"
+        gsutil -m cp -r "$OUTPUT/spraggins/spraggins.ims.zarr" "gs://$CLOUD_TARGET/spraggins/spraggins.ims.zarr"
         gsutil -m cp -r "$OUTPUT/linnarsson/linnarsson.images.zarr" "gs://$CLOUD_TARGET/linnarsson/linnarsson.images.zarr"
```

Comment:
This is starting to get cluttered. Two possible directions:
- Put this in a for-loop, like
```suggestion
        for DIR in vanderbilt/vanderbilt.images spraggins/spraggins.ims.zarr linnarsson/linnarsson.images.zarr; do
          gsutil -m cp -r "$OUTPUT/$DIR" "gs://$CLOUD_TARGET/$DIR"
        done
```
- or, perhaps better, look into [`gsutil rsync`](https://cloud.google.com/storage/docs/gsutil/commands/rsync), which does have an exclude option.